src/generic/delete-modal/DeleteModal.test.jsx

```diff
@@ -82,4 +82,17 @@ describe('<DeleteModal />', () => {
     fireEvent.click(cancelButton);
```

Comment:
Can we replace all `fireEvent` with `userEvents`?

Reply:
This does not apply to my PR

Reply:
Fixed